integration/src/main/resources/configs/perf/GetProviderPassport.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "GetProviderPassport",
+  "description": "Gets a user's passport from the specified provider.",
+  "serverSpecificationFile": "ecm-local.json",
+  "kubernetes": {},
+  "application": {},
+  "testScripts": [
+    {
+      "name": "GetProviderPassport",
+      "numberOfUserJourneyThreadsToRun": 1000,
```

Comment:
NB: This config gets close to what we want, but will probably need to be updated in the relatively near future as there is work upcoming from the PF team on better ramp-up load configs for TestRunner